docs/getting-started-v2-beta.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: page-with-side-nav
+title: Getting Started
+---
+# Getting Started
+
+De 'Bevraging Persoon' Web API is gespecificeerd in OpenAPI specifications (OAS).
+
+Wil je de API gebruiken? Dit kun je doen:
+
+1. Bekijk de [functionaliteit en specificaties](#functionaliteit-en-specificaties)
+2. [Implementeer de API Client](#implementeer-de-api-client)
+3. [Probeer en test de API](#probeer-en-test-de-api)
+
+## Functionaliteit en specificaties
+
+Je kunt een visuele representatie van de specificatie genereren met [Redoc]({{ site.baseurl }}/redoc).
+
+De (resolved) OAS3 kun je hier downloaden: [openapi.yaml](https://github.com/VNG-Realisatie{{ site.baseurl }}/blob/v2.0.0beta1/specificatie/genereervariant/openapi.yaml).
+
+De [functionele documentatie](./features-v2-beta) vind je in [features](./features-v2-beta).
+
+## Implementeer de API client
+Client code kun je genereren met de "[genereervariant](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v2.0.0beta1/specificatie/genereervariant/openapi.yaml){:target="_blank" rel="noopener"}" van de API-specificaties en een code generator. Een overzicht met codegeneratoren kun je vinden op [OpenAPI.Tools](https://openapi.tools/#sdk){:target="_blank" rel="noopener"}.
```

Comment:
Gebruik hier ook de notatie met {{ site.baseurl }}, dus:

https://github.com/VNG-Realisatie{{ site.baseurl }}/blob/v2.0.0beta1/specificatie/genereervariant/openapi.yaml

Dat vind ik wel een goede vondst want als we de repo nog eens hernoemen naar /Haal-Centraal-BRP-Persoon hoeven we alleen maar de variabele baseurl in de _config.yml aan te passen en werken alle links weer.

Reply:
aangepast